rakelib/utils.rake

```diff
@@ -68,6 +68,7 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
That's actually ok for me. It avoids weird hacks and development of something that might not be needed in a few weeks/months (hopefully).